CONTRIBUTING.md

```diff
@@ -10,9 +10,33 @@ Then, open your console on the project directory and run `bundle install` and `y
 
 Finally, run `rake assets:compile`.
 
-## Running Tests
+## Running tests
 This library has a Ruby component and a JavaScript component. Each component has its own test suite.
 
-To run both suites, open your console and run `rake test` from the project directory.
+To run both suites, plus an asset compilation check, open your console and run `rake test` from the project directory. Ruby tests will run first. You will then be prompted to open your web browser to run the JavaScript tests.
 
-Ruby tests will run first. You will then be prompted to open your web browser to run the JavaScript tests.
+To only run the JavaScript tests, open your console and run `yarn test` from the project directory.
+
+To only run the Ruby tests, open your console and run `rake test:helpers` from the project directory.
+
+## Testing specific time zones
+Stubbing the browser's time zone is not possible. Therefore, testing for a specific time zone needs to be done manually as follows:
+
+1. Build the project with `yarn build`.
+2. Run `yarn start`.
+3. Open http://localhost:9000/time-zone-check in your browser.
+4. Choose a location that observes the time zone you want to test for and add it to Chrome's [geolocation presets](https://developer.chrome.com/docs/devtools/settings/locations/).
+5. Use Chrome's [location sensor override feature](https://developer.chrome.com/docs/devtools/sensors/#open-sensors) to set your location to the one you chose in step 1.
+6. Refresh the page and visually verify the results.
+
+Here's a list of known edge-case time zones and their locations:
+
+TZ identifier     | Standard Abbreviation | DST Abbreviation | Latitude | Longitude
+----------------- | --------------------- | ---------------- | -------- | ---------
+Asia/Jakarta      | WIB                   | —                | -6.175   | 106.8275
+America/Anchorage | AKST                  | AKDT             | 61.2181  | -149.9003
+Europe/Paris      | CET                   | CEST             | 48.8576  | 2.3470
+Pacific/Honolulu  | HST                   | —                | 21.5376  | -158.0023
+Asia/Singapore    | SGT                   | —                | 1.3521   | 103.8198
+Europe/Moscow     | MSK                   | —                | 55.7558  | 37.6173
+Asia/Shanghai     | CST                   | —                | 31.2304  | 121.4737
```

Comment:
Great guide! Ideally the browser's Location settings would have an API we could use in our unit tests but this tooling is great even if we had that too.